data/src/main/java/com/canopas/yourspace/data/service/auth/AuthService.kt

```
@@ -101,16 +101,25 @@ class AuthService @Inject constructor(
+            locationManager.stopLocationTracking()
+            currentUser = null
+            currentUserSession = null
+            userPreferences.isFCMRegistered = false
+            userPreferences.setOnboardShown(false)
+            userPreferences.currentSpace = ""
+            userPreferences.clearPasskey()
+            userPreferences.clearPrivateKey()
+            firebaseAuth.signOut()
+            locationManager.stopService()
+            locationCache.clear()
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)